surefire-api/src/test/java/org/apache/maven/surefire/api/testset/TestListResolverTest.java

```diff
@@ -267,13 +270,16 @@ public void testShouldRunAny()
     public void testClassFilter()
     {
         TestListResolver resolver = new TestListResolver( "#test" );
-        assertTrue( resolver.shouldRun( "pkg/MyTest.class", null ) );
+        assertTrue( resolver.shouldRun( "pkg/MyTest.class", "test" ) );
```

Comment:
Don't change the assertions.
It make whole PR not trusty.

Reply:
try:
```
        resolver = new TestListResolver( "#abc" );
        assertTrue( resolver.shouldRun( "pkg/MyTest.class", null ) );

        resolver = new TestListResolver( "#test" );
        assertTrue( resolver.shouldRun( "pkg/Abc.class", null ) );
```
what we verify here?